service/controller/v16/key/key.go

```diff
@@ -506,6 +506,28 @@ func WorkerCount(customObject v1alpha1.AWSConfig) int {
 	return len(customObject.Spec.AWS.Workers)
 }
 
+// WorkerDockerVolumeSizeGB returns size of a docker volume configured for
+// worker nodes. If there are no workers in custom object, 0 is returned as
+// size.
+func WorkerDockerVolumeSizeGB(customObject v1alpha1.AWSConfig) (int, error) {
+	if len(customObject.Spec.AWS.Workers) <= 0 {
+		return 0, nil
+	}
+
+	v := customObject.Spec.AWS.Workers[0].DockerVolumeSizeGB
+
+	if v == "" {
+		return 0, nil
```

Comment:
Should it be 0 or 100?

Reply:
It looks counterintuitive to me. Is there a reason why it's like that? Can that be changed so we return the default here?

Reply:
+1 on defaulting in the key package. We do similar things here already.